src/ViewDirectives/WidgetDirective.php

```diff
@@ -30,8 +30,16 @@ function () use ($location, $type, $handle, $entities, $replace) {
                     $widgetClass = config('rapidez.widgets.'.$widget->instance_type);
 
                     if (!class_exists($widgetClass)) {
```

Comment:
Might it be an idea to extract all this logging to a different function?
It'll allow us to return early and hopefully reduce the amount of nesting